src/confluent_kafka/src/confluent_kafka.h

```diff
@@ -386,10 +386,11 @@ PyObject *c_Node_to_py(const rd_kafka_Node_t *c_node);
 rd_kafka_topic_partition_list_t *py_to_c_parts (PyObject *plist);
 PyObject *list_topics (Handle *self, PyObject *args, PyObject *kwargs);
 PyObject *list_groups (Handle *self, PyObject *args, PyObject *kwargs);
-
+PyObject *set_sasl_credentials(Handle *self, PyObject *args, PyObject *kwargs);
 
```

Comment:
change in styling. may be should add a new line.

Reply:
done